public/docs/ts/latest/guide/router.jade

```diff
@@ -401,7 +404,7 @@ h4#register-providers Register routing in bootstrap
   Providing the router providers at the root makes the Router available everywhere in our application.
```

Comment:
Not precisely from this update. We normally call "root" to the AppComponent and not the bootstrap. Wouldn't be better to say "providers at boostrap makes the ..."